dev/reference/key-monitoring-metrics/tikv-dashboard.md

```diff
@@ -18,184 +18,86 @@ To understand the key metrics displayed on the Overview dashboard, check the fol
-Cluster | IO utilization | The I/O utilization of each TiKV instance |
-Cluster | MBps | The total bytes and keys of read and write on each TiKV instance |
-Cluster | QPS | The QPS of different kinds of commands in each TiKV instance |
-Cluster | Errps | The total number of the gRPC message failures |
-Cluster | Leader | The number of leaders on each TiKV instance |
-Cluster | Region | The number of Regions on each TiKV instance |
-Errors | Server is busy | It contains some kinds of events such as write stall, channel full, scheduler busy, and coprocessor full, which will make the TiKV instance unavailable temporarily. |
-Errors | Server report failures | The total number of reporting failure messages | It should be `0` in normal case.
-Errors | Raftstore error | The number of different raftstore errors on each TiKV instance |
-Errors | Scheduler error | The number of different scheduler errors on each TiKV instance |
-Errors | Coprocessor error | The number of different coprocessor errors on each TiKV instance |
-Errors | gRPC message error | The number of different gRPC message errors on each TiKV instance |
-Errors | Leader drop | The count of dropped leader in each TiKV instance |
-Errors | Leader missing | The count of missing leader in each TiKV instance |
-Server | Leader | The number of leaders on each TiKV instance |
-Server | Region | The number of Regions on each TiKV instance |
-Server | CF size | The total size of each column family |
-Server | Store size | The storage size of each TiKV instance |
-Server | Channel full | The total number of channel full errors on each TiKV instance | It should be `0` in normal case.
-Server | Server report failures | The total number of reporting failure messages |
-Server | Region average written keys | The average rate of writing keys to Regions on each TiKV instance |
-Server | Region average written bytes | The average rate of writing bytes to Regions on each TiKV instance |
-Server | Active written leaders | The number of active leaders on each TiKV instance |
+Cluster | Store size | The storage size per TiKV instance |
+Cluster | Available size | The available capacity per TiKV instance |
+Cluster | Capacity size | The capacity size per TiKV instance |
+Cluster | CPU | The CPU usage per TiKV instance |
+Cluster | Memory | The memory usage per TiKV instance |
+Cluster | IO utilization | The I/O utilization per TiKV instance |
+Cluster | MBps | The total bytes of read and write in each TiKV instance |
+Cluster | QPS | The QPS per command in each TiKV instance |
+Cluster | Errors-gRPC | The total number of gRPC message failures |
+Cluster | Leaders | The number of leaders per TiKV instance |
+Cluster | Regions | The number of Regions per TiKV instance |
+Errors | Server is busy | Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Write Stall, Channel Full, Scheduler Busy, and Coprocessor Full
+ |
```

Comment:
It looks like this line will lead to a [render error](https://github.com/pingcap/docs/blob/aefa5b611d4d3efa58c796cc02ca665ffbdb6673/dev/reference/key-monitoring-metrics/tikv-dashboard.md) in github.

![Screenshot_2019-09-05_13-15-41](https://user-images.githubusercontent.com/5244316/64313767-514a6f00-cfdf-11e9-96d6-d7518dd47eb4.png)